Web preview: remove fake placeholder in favour of <Spinner>.

mtias opened the pull request:
We had previously discussed the fake placeholders were not the best solution since they won't conform at all to the theme. This replaces it with a `<Spinner>`. (Also removes the placeholder message `prop` since it's no longer needed.)

timmyc commented on client/post-editor/editor-preview/index.jsx:
```diff
@@ -86,7 +86,6 @@ const EditorPreview = React.createClass( {
 				defaultViewportDevice="tablet"
 				onClose={ this.props.onClose }
 				previewUrl={ this.state.iframeUrl }
-				loadingMessage="Beep beep boop…"
```
:(

mtias replied:
We need to find a better place for it :)

timmyc commented:
This is working well for me in Chrome - in mobile Safari however I do not see the spinner at all in the preview window.

nylen commented:
Why remove the placeholder message here?  From pre-oss discussion:

> If we want do to a "Beep beep boop" easter egg, an idea: add the pulsating dot, and just below it, "Beep beep book" appear like a shadow (fade in, stay for a second, fade out) in light gray color. ;)

folletto commented:
Can we use the pulsating dot instead of that spinner?

mtias commented:
Restored message:

![image](https://cloud.githubusercontent.com/assets/548849/11682946/1b66965e-9e68-11e5-947a-c1189ea5c85c.png)

@folletto I tried pulsing dot but it didn't quite work, too small, the colors are hard set, etc.

mtias commented:
@timmyc it didn't appear on mobile because web-preview had the wrong width on those sizes. I've fixed that and removed the toggle buttons on mobile since they don't do anything really.

Mind giving another look?

folletto commented:
> I tried pulsing dot but it didn't quite work, too small, the colors are hard set, etc.

I guess we have to create a new loader component then ;)

mtias commented:
@folletto yep, another iteration on pulsing dot probably, and document it.

timmyc commented:
LGTM :shipit:
